src/site/stages/build/process-cms-exports/transformers/paragraph-wysiwyg.js

```diff
@@ -0,0 +1,21 @@
+const { getDrupalValue } = require('./helpers');
+
+function wysiwygTransform(entity) {
+  const { fieldWysiwyg, contentModelType } = entity;
+
+  const transformed = {
+    contentModelType,
```

Comment:
Since this is required for the validation we may want to move this responsibility out of each specific transformer

Reply:
Addressed with: https://github.com/department-of-veterans-affairs/vets-website/pull/11078/commits/457dda8d1ed4b09e19911876969175aec3e75824